src/utils/_opaciy.scss

```diff
@@ -0,0 +1,14 @@
+// --- Opacity --- //
+
+// alpha parameter for rgba Colors
+// @include bgcolor-opacity($c-black);
+
+$opacity-value: (15, 25, 50, 75, 100) !default;
```

Comment:
Is it validated with UX? Maybe we should move it to settings. Not sure if utils should define variables within their files.

Reply:
Yes, it's validated, ok you're right, I move it to settings